src/herder/HerderPersistenceImpl.cpp

```diff
@@ -230,6 +236,10 @@ HerderPersistence::copySCPHistoryToStream(Database& db, soci::session& sess,
                                           XDROutputFileStream& scpHistory)
 {
     ZoneScoped;
+    // TODO: this may conflict with main thread, as this is done in the
+    // background (this is the case in master today, so can be fixed
```

Comment:
Was this supposed to be 'this is not the case in master'? Or am I missing something?

Reply:
What I meant to say here is that master today already has this problem, I opened https://github.com/stellar/stellar-core/issues/4589 to track it. So I can either remove this TODO, or put the issue link in the comment. Wdyt?

Reply:
Ahh, I see now. I think the way the comment is phrased is weird - when this is merged, this comment will become 'master today', so it will be kind of meaningless. I think it would be appropriate to state that this hasn't caused trouble for years and link to the issue maybe.